libknet/transport_udp.c

```diff
@@ -363,7 +364,7 @@ static int read_errs_from_sock(knet_handle_t knet_h, int sockfd)
 						case SO_EE_ORIGIN_ICMP:  /* ICMP */
 						case SO_EE_ORIGIN_ICMP6: /* ICMP6 */
 							origin = (struct sockaddr_storage *)(void *)SO_EE_OFFENDER(sock_err);
-							if (knet_addrtostr(origin, sizeof(origin),
+							if (knet_addrtostr(origin, sizeof(*origin),
```

Comment:
Why does `knet_addrtostr()`  take that size parameter? It does not use it (except for returning `EINVAL` if it's zero):
https://github.com/kronosnet/kronosnet/blob/69797cb15138beb87f3afdae2cd9cd74626f7dca/libknet/netutils.c#L142-L176

Reply:
IIRC originally it was used in place of sockaddr_len(ss) and it´s probably just a leftover that could be removed.